cms/templates/index.html

```diff
@@ -227,10 +227,11 @@ <h3 class="course-title">${course_info['display_name'] | h}</h3>
                 </div>
 
                 <dl class="course-status">
-                  <dt class="label sr">This re-run processing status:</dt>
+                  ## Translators: This is a status message for the course re-runs feature. When a course admin indicates that a course should be re-run, the system needs to process that and get the new course re-run ready to go. The status of that re-run process will follow this text.
```

Comment:
Also the last part of this is confusing. Suggest rewording:

"the system needs to process the request and prepare the new course. The status of the process will follow this text."